frontend/packages/dev-console/src/components/topology/helm-details/TopologyHelmReleasePanel.tsx

```diff
@@ -27,6 +56,10 @@ const TopologyHelmReleasePanel: React.FC<TopologyHelmReleasePanelProps> = ({ hel
       )
     : navFactory.details(HelmReleaseOverview).component;
 
+  const resourcesComponent = manifestResources
+    ? () => <TopologyHelmReleaseResourcesPanel manifestResources={manifestResources} />
+    : () => <></>;
```

Comment:
Can't we return null here? `() => null`

Reply:
I'm actually surprised that is not a linter error. An empty fragment?

cc @christianvogt

Reply:
Agree with the suggested change.
If we can find a lint rule for it. Enable it.

Reply:
Fixed. I didn't see an available lint rule for it.